scripts/run-integration-tests.sh

```diff
@@ -1,31 +1,32 @@
 #!/bin/bash
 
 # install monero and run daemon and wallet RPC servers for alice and bob
-bash ./scripts/install-monero-linux.sh
+./scripts/install-monero-linux.sh
 echo "starting monerod..."
-./monero-x86_64-linux-gnu-v0.17.3.2/monerod --detach --regtest --offline --fixed-difficulty=1 --rpc-bind-port 18081 &
+./monero-bin/monerod --detach --regtest --offline --fixed-difficulty=1 --rpc-bind-ip 127.0.0.1 --rpc-bind-port 18081
 sleep 5
 
 echo "starting monero-wallet-rpc on port 18083..."
 mkdir bob-test-keys
-./monero-x86_64-linux-gnu-v0.17.3.2/monero-wallet-rpc --rpc-bind-port 18083 --disable-rpc-login --wallet-dir ./bob-test-keys &> monero-wallet-cli-bob.log &
+./monero-bin/monero-wallet-rpc --rpc-bind-ip 127.0.0.1 --rpc-bind-port 18083 --disable-rpc-login --wallet-dir ./bob-test-keys &> monero-wallet-cli-bob.log &
 MONERO_WALLET_CLI_BOB_PID=$!
 
 sleep 5
 curl http://localhost:18083/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"create_wallet","params":{"filename":"test-wallet","password":"","language":"English"}}' -H 'Content-Type: application/json'
+echo
 
 echo "starting monero-wallet-rpc on port 18084..."
 mkdir alice-test-keys
-./monero-x86_64-linux-gnu-v0.17.3.2/monero-wallet-rpc --rpc-bind-port 18084 --disable-rpc-login --wallet-dir ./alice-test-keys &> monero-wallet-cli-alice.log &
+./monero-bin/monero-wallet-rpc --rpc-bind-ip 127.0.0.1 --rpc-bind-port 18084 --disable-rpc-login --wallet-dir ./alice-test-keys &> monero-wallet-cli-alice.log &
 MONERO_WALLET_CLI_ALICE_PID=$!
 
 # install ganache and run 
 echo "installing and starting ganache-cli..."
-if ! command -v golangci-lint &> /dev/null; then
-	npm i -g ganache-cli
+if ! command -v ganache-cli &> /dev/null; then
+	echo "Please install ganache-cli and place it in your path"
 fi
```

Comment:
I made this same change in 3 different scripts.  The check was looking at `golangci-lint` instead of `ganache-cli` and the nmp install will fail unless you have write access to /usr/local/*.

Any opinion on how to better handle the error?  I could add an `exit` after the error message, but to we'd need start `ganache-cli` before any of the other daemons to make sure any started daemons get cleaned up.

Reply:
good catch! can you re-add the `npm i -g ganachi-cli` line? the CI now fails as it doesn't have ganache-cli installed. alternatively I think you can update the CI files to install ganache-cli there.

Reply:
I added it back and I also added instructions to `docs/local.md` on how to set the install prefix to a directory that doesn't require root.  (For anyone else like myself that installed node with `snap` on Ubuntu.)